app/build.gradle

```diff
@@ -1,25 +1,29 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
-apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
     defaultConfig {
-        applicationId "com.plangrid.skeleton"
+        applicationId "com.plangrid.plansensor"
```

Comment:
should probably keep this part as plansensor, but not a huge deal.

Reply:
oooh yep, willdo